src/SFA.DAS.Assessor.Functions.ExternalApis/Assessor/Types/ImportLearnerDetailResponse.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
```

Comment:
The learner details parts aren't used currently they are part of the DC API work